src/medication-request/medication-request.resource.tsx

```diff
@@ -19,7 +19,7 @@ export function usePrescriptionsTable(
   pageOffset: number = 0,
   patientSearchTerm: string = ""
 ) {
-  const { data, error } = useSWR<{ data: EncounterResponse }, Error>(
+  const { data, mutate, error } = useSWR<{ data: EncounterResponse }, Error>(
```

Comment:
I created a mutate element at this level so we can update the prescription table row when a dispense form is saved